packages/contracts/scripts/deploy/manager/DiamondDeployHelper.ts

```diff
@@ -0,0 +1,40 @@
+import { ethers } from "ethers";
+import path from "path";
+
+export const FacetCutAction = { Add: 0, Replace: 1, Remove: 2 };
+
+export async function getSelectorsFromFacet(contractName: string, artifactFolderPath = "../../../out") {
+    const contractFilePath = path.join(artifactFolderPath, `${contractName}.sol`, `${contractName}.json`);
+    const contractArtifact = require(contractFilePath);
+    const abi = contractArtifact.abi;
+    const bytecode = contractArtifact.bytecode;
+    const target = new ethers.ContractFactory(abi, bytecode);
+    const signatures = Object.keys(target.interface.functions);
+
+    const selectors = signatures.reduce((acc, val) => {
+        if (val !== "init(bytes)") {
+            // @ts-ignore
+            acc.push(target.interface.getSighash(val));
+        }
+        return acc;
+    }, []);
+
+    console.log(`Selectors of ${contractName}: `, selectors);
+
+    return selectors;
+}
+
+export function getContractInstance(contractName: string, account?: any, artifactFolderPath = "../../../out") {
```

Comment:
I don't have the exact answer off hand, but I feel that using relative paths like this is very fragile for if we move files and directories around. Perhaps you could take a moment to think of a more robust implementation utilizing `path.join` or `path.resolve`?

Reply:
> Perhaps you could take a moment to think of a more robust implementation utilizing `path.join` or `path.resolve`?

Yeah, I've already thought about it. But as for my experience so far, seems there is no more robust implementation with `path.join`, I guess

Reply:
> Perhaps you could take a moment to think of a more robust implementation utilizing `path.join` or `path.resolve`?

Anyways, will think more about it

Reply:
> Perhaps you could take a moment to think of a more robust implementation utilizing `path.join` or `path.resolve`?

[This module](https://www.npmjs.com/package/app-root-path) seems good for getting app root path, shall we use this module? @pavlovcik